modules/preview-react/status-indicator/lib/StatusIndicatorLabel.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {Text, TextProps} from '@workday/canvas-kit-react/text';
+
+export interface StatusIndicatorLabelProps extends TextProps {}
+
+export const StatusIndicatorLabel = createComponent('span')({
+  displayName: 'StatusIndicator.Label',
+  Component: ({...elemProps}: StatusIndicatorLabelProps, ref, Element) => {
+    return (
+      <Text
+        whiteSpace="nowrap"
+        overflow="hidden"
+        textOverflow="ellipsis"
+        typeLevel="subtext.large"
+        fontWeight="bold"
+        textTransform="capitalize"
+        ref={ref}
+        color="inherit"
+        as={Element}
+        {...elemProps}
+      >
+        {elemProps.children}
```

Comment:
why did you use `elemProps.children` instead of just `children`? I guess you can destructure children from `elemProps`.